spec/spec_helper.rb

```diff
@@ -80,6 +100,11 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.before type: :system do
+    # Override the default driver used by rspec system tests
+    driven_by :selenium_remote_chrome
```

Comment:
While I have moved what driver configuration I could globally, according to the rspec [system test documentation](https://relishapp.com/rspec/rspec-rails/v/5-1/docs/system-specs/system-spec) this `driven_by :selenium_remote_chrome`
setting is still needed to override the default driver
As mentioned [here](https://github.com/rails/rails/issues/34379) this behaviour is intended and I don't think is likely to change.

If we wanted this configured globally too, using the older [feature type](https://relishapp.com/rspec/rspec-rails/v/5-1/docs/feature-specs/feature-spec) rspec tests I think would work though my understanding is that it has slightly less functionality than system tests.